tip-wlan/resources/environments/dev-amazon-tip.yaml

```diff
@@ -120,6 +120,11 @@ wlan-port-forwarding-gateway-service:
   enabled: true
   creds:
     websocketSessionTokenEncKey: MyToKeN0MyToKeN1
+  env:
+    pfGateway:
+    externallyVisibleHost: api.wlan.demo.lab.wlan.tip.build
```

Comment:
Indentation needs fixing or is it just this tool?

Reply:
The indentation needs fixing. Thank you very much! I will double-check the indentations for other files too